operators/pkg/controller/kibanaassociation/ca.go

```diff
@@ -31,27 +31,27 @@ func CACertSecretName(kibanaName string) string {
 // The CA secret content is copied over from ES CA secret into a dedicated secret for Kibana.
 func (r *ReconcileAssociation) reconcileCASecret(kibana kbtype.Kibana, es types.NamespacedName) (string, error) {
```

Comment:
Should we rename this? We don't really reconcile a CA anymore.

Reply:
Same as for APM, for Kibana this is the `elasticsearch.ssl.certificateAuthorities` config, so CA seems to be a valid naming scheme